graylog2-web-interface/src/views/components/widgets/MessageList.jsx

```diff
@@ -194,4 +159,11 @@ export default connect(MessageList,
   {
     selectedFields: SelectedFieldsStore,
     currentView: ViewStore,
-  });
+    searches: SearchStore,
+  }, props => Object.assign(
+    {},
+    props,
+    {
+      effectiveTimerange: props.searches.result.results[props.currentView.activeQuery].effectiveTimerange,
```

Comment:
We need the effective timerang range to "freeze" the search when using the pagination.
I would like to implement a cleaner solution for the above row. I thought about extracting the effective timerange in the ExtendedSearchPage, because we are already connecting the SearchStore there. But I am not a fan of passing the attribute through all the components in between.

I also thought about passing the full results objects to the MessageList component to extract the attribute there in a safer way. but I prefer to pass only the needed attributes to the component.

Anyone has some thoughts on this?